configure.ac

```diff
@@ -1,5 +1,13 @@
-AC_PREREQ([2.60])
-AC_INIT([libsecp256k1],[0.1])
+AC_PREREQ([2.69])
+
+define(_LIBSECP256K1_VERSION_MAJOR, 0)
+define(_LIBSECP256K1_VERSION_MINOR, 8)
```

Comment:
Should be 99 according to the preliminary plan (https://github.com/bitcoin-core/secp256k1/issues/856). I didn't have a particular reason for this besides matching Bitcoin Core's approach. But if I understand this PR correctly , `secp256k1_version` doesn't include `_LIBSECP256K1_VERSION_IS_RELEASE`. If minor version was 99 for non-releases, `secp256k1_version` can be used to distinguish between releases and non-releases.

Reply:
Yeah - I picked 0.8 to provide additional headroom for breaking API changes before stabilization.

0.8 could be cut as a branch now-ish

0.9 would represent a nearly complete API and allow for stabilization from 0.9 -> 0.9.9

and 1.0 would reflect final API / ABI